src/conf_mode/protocols_static_multicast.py

```diff
@@ -86,10 +86,10 @@ def verify(mroute):
     if mroute is None:
         return None
 
-    for route in mroute['mroute']:
-        route = route.split('/')
+    for mroute in mroute['mroute']:
```

Comment:
Perhaps another name than 'mroute' should be used? It is overriding the function parameter.